### PR TITLE
Renaming mysql service instance attrs

### PR DESCRIPTION
**Pull request by mbfrahry**
This PR swaps some of the attributes and corresponding names around. This hasn't been released yet so there won't be any breaking changes

### CODE REVIEW

**Comment by mbfrahry** [comment]
Passing tests 
image

File: oraclepaas/resource_mysql_service_instance.go [hunk 1]
```diff
@@ -72,12 +72,6 @@ func resourceOraclePAASMySQLServiceInstance() *schema.Resource {
 				}, true),
 			},
 
-			"enable_notification": {
-				Type:     schema.TypeBool,
-				Optional: true,
-				ForceNew: true,
-			},
```
Comment (tombuildsstuff) [inline oraclepaas/resource_mysql_service_instance.go]: given this is defaulted below, I'm assuming we don't want to parse it out?

File: oraclepaas/resource_mysql_service_instance.go [hunk 2]
```diff
@@ -106,6 +100,19 @@ func resourceOraclePAASMySQLServiceInstance() *schema.Resource {
...
+				Type:     schema.TypeString,
+				Optional: true,
+				ForceNew: true,
+			},
+
+			"shape": {
+				Type:     schema.TypeString,
+				Optional: true,
+				ForceNew: true,
+				Default:  "oc3",
```
Comment (tombuildsstuff) [inline oraclepaas/resource_mysql_service_instance.go]: is that a sensible default?

File: website/docs/r/oraclepaas_mysql_service_instance.html.markdown [hunk 3]
```diff
@@ -67,12 +68,14 @@ The following arguments are supported:
 
 * `availability_domain` - (Optional) Name of the availability domain within the region where the Oracle Database Cloud Service instance is to be provisioned. This is applicable only if you wish to provision to an OCI instance.
 
-* `enable_notification` - (Optional) Specifies if you would like to receive notification emails. The default is `true`
-
 * `notification_email` - (Optional) The email address to send notifications around successful or unsuccessful completions of the instance-creation operation.
 
 * `ip_network` - (Optional) This attribute is only applicable to accounts where regions are supported. The three-part name of an IP network to which the service instance is added. For example: /Compute-identity_domain/user/object
 
+* `shape` - (Optional) The desired compute shape.  A shape defines the number of Oracle Compute Units (OCPUs) and amount of memory (RAM). See [About Shapes](http://www.oracle.com/pls/topic/lookup?ctx=cloud&id=OCSUG210) in _Using Oracle Compute Cloud Service_ for more information about shapes.
```
Comment (tombuildsstuff) [inline website/docs/r/oraclepaas_mysql_service_instance.html.markdown]: `Defaults to [value]`

File: oraclepaas/resource_mysql_service_instance.go (hunk 2 above)
Comment (scross01) [reply oraclepaas/resource_mysql_service_instance.go]: I'd recomment removing the default, so the user must specify it explictly

File: oraclepaas/resource_mysql_service_instance.go (hunk 1 above)
Comment (mbfrahry) [reply oraclepaas/resource_mysql_service_instance.go]: If somebody is specifying a `NotificationEmail` than they have to set `EnableNotification` to `true`. You can't set `EnableNotification` without an email so we are just setting the value in terraform so the user doesn't have to

File: website/docs/r/oraclepaas_mysql_service_instance.html.markdown (hunk 3 above)
Comment (mbfrahry) [reply website/docs/r/oraclepaas_mysql_service_instance.html.markdown]: Taking @scross01 comment above to remove the default and make this required

File: oraclepaas/resource_mysql_service_instance_test.go [hunk 4]
```diff
@@ -209,18 +209,18 @@ resource "oraclepaas_mysql_service_instance" "test" {
...
 	mysql_configuration = {
 		db_name			= "demo_db"
 		db_storage      = 25
 		mysql_port      = 3306
 		mysql_username  = "root"
 		mysql_password  = "MySqlPassword_1"
-		shape           = "oc3"
 		mysql_charset   = "utf8"
 		mysql_collation = "utf8_general_ci"	    
 		subnet          = "%s"
```
Comment (scross01) [inline oraclepaas/resource_mysql_service_instance_test.go]: `subnet` should also be moved up

**Pull request merged**